tests/lib/rules/prefer-find-by.test.ts

```diff
@@ -149,6 +149,14 @@ ruleTester.run(RULE_NAME, rule, {
         })
       `,
     },
+    {
+      code: `
+        import {screen, waitFor} from '@testing-library/foo';
+        it('tests', async () => {
+          await waitFor(() => expect(screen.querySelector('baz')).toBeInTheDocument());
```

Comment:
If you check the original issue #501, the problematic code was a bit different:

```js
    it('renders', async () => {
      const { container } = render(<App />)

      await waitFor(() =>
        expect(container.querySelector("[data-icon-name='bookmark-filled']")).toBeInTheDocument(), // reported
      )
    })
```

Can you add a test for this one too, please?

Reply:
I'll add this test, I missed that the original issue was slightly different to what I was experiencing on my side. :+1: